tests/unit/test_utils.py

```diff
@@ -74,30 +74,30 @@ def test_map_dl_vlan(self):
         "vlan_range,expected",
         [
             (
-                [101, 200],
+                [[101, 200]],
                 [
-                    (101, 4095),
-                    (102, 4094),
-                    (104, 4088),
-                    (112, 4080),
-                    (128, 4032),
-                    (192, 4088),
-                    (200, 4095),
+                    "101/4095",
+                    "102/4094",
+                    "104/4088",
+                    "112/4080",
+                    "128/4032",
+                    "192/4088",
+                    "200/4095",
                 ]
             ),
             (
-                [101, 90],
+                [[101, 90]],
                 []
             ),
             (
-                [34, 34],
-                [(34, 4095)]
+                [[34, 34]],
```

Comment:
For completeness since it's accepting a range, can you also cover with more than one element, such as: `[[34, 34], [128, 128], [130, 135]]`?

Reply:
Added test